setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+import madprops
+
+
+setup(
+    name='drf-madprops',
```

Comment:
We might have something besides serializer in this lib, so I tried to make it more generic.